Nodejs/Product/Nodejs/Templates/Files/TypeScriptTsConfig/tsconfig.json

```diff
@@ -8,6 +8,12 @@
     "target": "es5"
   },
   "exclude": [
-    "node_modules"
+    "**/bin",
```

Comment:
I'm not sure if all the directories are relevant for this kind of project, but I think is still a good idea to include them as a general list of excluded. Let me know if you think this should be removed.

Reply:
I don't think we should change the tsconfig.json template. People don't keep TypeScript source files under those folders.

Reply:
Ok, rollback.